src/index.js

```diff
@@ -24,7 +24,7 @@ const fetchScores = async () => {
     const { user, score } = item;
     return `<div class='score-wrapper'>
               <div class='score'>
-                <p>${user}:</p><p>${score}</p>
+                <p>${user}</p><p>: ${score}</p>
               </div>
```

Comment:
- [ ] [OPTIONAL] I think it would be better to sort the score short to big like , " 1,2,3,55,60....." 😉 

![image](https://user-images.githubusercontent.com/75126481/218059468-bb55a393-c581-4334-ba46-4acbc91090df.png)